### PR TITLE
XW-3127 | new right rail recirculation

### PR DESCRIPTION
**Pull request by latata**
For the test I just copied rail recirc module code. We gonna clean it up when we will productize it.

https://wikia-inc.atlassian.net/browse/XW-3127

@Wikia/x-wing

### CODE REVIEW

File: extensions/wikia/Recirculation/js/views/premiumRail.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+		return curated.injectContent(data)
+			.then(renderTemplate('client/premiumRail.mustache'))
+			.then(utils.waitForRail)
+			.then(function ($html) {
+				if (options.before) {
+					$html = options.before($html);
+				}
+
+				$('#RECIRCULATION_RAIL').html($html);
```
Comment (rybmat) [inline extensions/wikia/Recirculation/js/views/premiumRail.js:24]: why this id is written with ALL_CAPS?

File: extensions/wikia/Recirculation/styles/premium-rail.scss [hunk 2]
```diff
@@ -0,0 +1,64 @@
+@import 'skins/shared/color';
+@import 'skins/shared/mixins/transform';
+@import 'skins/shared/mixins/flexbox';
+
+.WikiaRail .premium-recirculation-rail {
+	border: 0;
```
Comment (rybmat) [inline extensions/wikia/Recirculation/styles/premium-rail.scss:6]: is there any difference between border: 0; and border: none; ?

**Review by rybmat — APPROVED** [review]
two minor comments but lgtm

File: extensions/wikia/Recirculation/styles/premium-rail.scss (hunk 2 above)
Comment (latata) [reply extensions/wikia/Recirculation/styles/premium-rail.scss:6]: It's shorter ;)

File: extensions/wikia/Recirculation/js/views/premiumRail.js (hunk 1 above)
Comment (latata) [reply extensions/wikia/Recirculation/js/views/premiumRail.js:24]: let's leave it for now

File: extensions/wikia/Recirculation/styles/premium-rail.scss (hunk 2 above)
Comment (weronika) [reply extensions/wikia/Recirculation/styles/premium-rail.scss:6]: and we should use border: 0; according to our css coding standards

File: extensions/wikia/Recirculation/templates/client/premiumRail.mustache [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+						{{#thumbnail}}
+							<img src="{{thumbnail}}" alt="{{title}}" itemprop="thumbnail">
+						{{/thumbnail}}
+					</div>
+					<div class="content">
+						<div class="title" title="{{title}}">{{title}}</div>
+
+							<div class="recirculation-sponsored-footer">
+								Sponsored by {{presented_by}}
+							</div>
```
Comment (weronika) [inline extensions/wikia/Recirculation/templates/client/premiumRail.mustache]: Do we alway have `presented_by` var set? And why every second item has this div hidden? I would also recommend to use i18n string here (even if not translated).

**Review by weronika — CHANGES_REQUESTED** [review]

File: extensions/wikia/Recirculation/templates/client/premiumRail.mustache (hunk 3 above)
Comment (latata) [reply extensions/wikia/Recirculation/templates/client/premiumRail.mustache]: That was just a temporary change for design review. Sorry I though that no one will review the code at the night. ;) I'd extract strings to i18n in recirc productization task.

**Review by weronika — APPROVED** [review]

**Pull request merged**